From 79d804defcf8176f5df58bf184b89a8021a7c15d Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Wed, 11 Mar 2020 09:03:37 +0800
Subject: [PATCH] video/rockchip: mpp: fix reg_id to int from u32

VPU without hardware id register will be set with -1 for reg_id,
so fix the reg_id from u32 to int.

Change-Id: Ic3cf15f223610428f77fd5e3291e74974a2a7981
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 drivers/video/rockchip/mpp/mpp_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/mpp/mpp_common.h b/drivers/video/rockchip/mpp/mpp_common.h
index 58271b8eb29b..d421763ceed2 100644
--- a/drivers/video/rockchip/mpp/mpp_common.h
+++ b/drivers/video/rockchip/mpp/mpp_common.h
@@ -128,7 +128,7 @@ struct mpp_hw_info {
 	/* register number */
 	u32 reg_num;
 	/* hardware id */
-	u32 reg_id;
+	int reg_id;
 	u32 hw_id;
 	/* start index of register */
 	u32 reg_start;
-- 
2.35.3

